test/internet/test-dgram-broadcast-multi-process.js

```diff
@@ -84,14 +84,16 @@ if (process.argv[2] !== 'child') {
       worker.messagesReceived = [];
 
       // Handle the death of workers
-      worker.on('exit', function(code, signal) {
+      worker.on('exit', (code, signal) => {
         // Don't consider this the true death if the worker
         // has finished successfully
         // or if the exit code is 0
         if (worker.isDone || code === 0) {
           return;
         }
 
+        assert.notStrictEqual(signal, null);
```

Comment:
Perhaps move this a few lines down to after the `console.error()` call so we get some debug in the test runner output?

Reply:
Good idea. Done!